src/account/__tests__/accountsReducer-test.js

```diff
@@ -16,16 +16,15 @@ import * as eg from '../../__tests__/lib/exampleData';
 
 describe('accountsReducer', () => {
   describe('REALM_ADD', () => {
-    const accountWithZulipVersion = eg.makeAccount({
+    const account1 = eg.makeAccount({
```

Comment:
I think changing `accountWithZulipVersion/accountWithoutZulipVersion` to a more general name `account1/account2` is better in terms of readability and coherence. Please tell me if you don't like this idea. :slightly_smiling_face:

Reply:
I like it!